utilities/transactions/optimistic_transaction_test.cc

```diff
@@ -719,7 +719,6 @@ TEST_P(OptimisticTransactionTest, ColumnFamiliesTest) {
   ASSERT_TRUE(s.IsBusy());
   s = txn_db->Get(read_options, handles[1], "AAAZZZ", &value);
   ASSERT_TRUE(s.IsNotFound());
-  ASSERT_EQ(value, "barbar");
```

Comment:
Note: this is most likely a typo in the original test that used to pass accidentally. The new code guarantees that after a queries returns `NotFound`, `value` will be empty.